advanced_topics/advanced-topics.md

```diff
@@ -1,6 +1,13 @@
-# Advanced Topics {#advanced-topics}
+---
+title: Advanced Topics
+has_children: true
```

Comment:
Is this true because this section will grow to have subsections? Otherwise, LGTM.

Reply:
yeah potentially